CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 0.1.3 - Apr 30, 2025 - "Routing Unicode in 2025"
+
+## Bug Fixes
+ * `Event::InvoiceRecieved` is now only generated once for each `Bolt12Invoice`
```

Comment:
PR desc says it is specifically when `manually_handle_bolt12_invoice` is enabled. But statement as it is true too then.

Reply:
Recieved -> Received

Reply:
Trying to make the best of a changelog review...